Dockerfile

```diff
@@ -0,0 +1,40 @@
+#maintainer Ariella Gladstein
+#organization tskit-dev
+#application "Msprime: A reimplementation of Hudson's classical ms simulator for modern data sets."
+
+FROM ubuntu:18.04
+
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy the current directory contents into the container at /app
+COPY . /app
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+            python3 \
+            python-dev \
+            python3-dev \
+            python-pip \
+            python3-pip \
+            build-essential \
+            libgsl0-dev \
+            libssl-dev \
```

Comment:
Do we need these libs below other than gsl0-dev? 

Also, I think we can use ``libgsl-dev`` these days

Reply:
Just `libgsl-dev` worked. Thanks.